acos_client/v30/session.py

```diff
@@ -24,8 +34,33 @@ def __init__(self, client, username, password):
 
     @property
     def id(self):
-        if self.session_id is None:
+        deviceIP = self.client.host
```

Comment:
The redis session context needs to be it's own class, else we're breaking the API for  everyone else that uses it.